pkg/kindsys/report.go

```diff
@@ -54,17 +73,53 @@ var plannedCoreKinds = []string{
 	"QueryHistory",
 }
 
+type Kind struct {
+	kindsys.SomeKindProperties
+	Category  string
+	SchemaRef string
+	GoRef     string
+	TsRef     string
+	DocsRef   string
```

Comment:
I'd be happy to move these into a: `links: { "go": ..., "docs": ....}` (or `refs` or anything similar).

Do you think that'd be better? @sdboyer 
If so, do you want me do it before getting this merged? Wouldn't be a big deal!

Reply:
Also related, do you want me to add the "grokRef" equivalent, pointing to code present [here](https://github.com/grafana/grok/tree/main/go/kinds)?

Same question, in this PR?

Reply:
i'd say i have a mild preference for keeping links grouped together, but no strong feeling :)

and yes let's definitely add grok links - that's where we have to start accounting for versioning. However, probably easier to do those in a follow-up, as you should sync with @malcolmholmes first on exactly what's worth linking to